chartjs-plugin-annotation.js

```diff
@@ -1,7 +1,7 @@
 /*!
  * chartjs-plugin-annotation.js
  * http://chartjs.org/
- * Version: 0.5.3
+ * Version: 0.5.4
```

Comment:
Please revert all changes to this file, we will build it during the release process.